src/components/fee-summary/FeeSummary.stories.ts

```diff
@@ -1,7 +1,11 @@
+import Vue from 'vue'
 import { FeeSummary } from './index'
 import { FilingCodes, CorpTypeCd } from '@bcrs-shared-components/enums/'
 import { FilingDataIF } from '@bcrs-shared-components/interfaces/'
 
+// for SbcFeeSummary
+Vue.filter('currency', x => `$${x}`)
```

Comment:
This fixes a runtime error.

Unfortunately this story still doesn't work because it's calling Pay API, which is returning a 401 (unauthorized). I tried to mock the axios call as we do in StaffComment, and tried using package `storybook-addon-mock`, but neither worked right away so I stopped spending time on it.

Reply:
Sounds good, no rush on this anywho.

Reply:
https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13734